features/step_definitions/reek_steps.rb

```diff
@@ -11,69 +11,72 @@
 end
 
 Then /^it reports nothing$/ do
-  assert_exact_output '', all_stdout
+  expect(last_command_started).to have_output_on_stdout('')
```

Comment:
Yes, it is probably right. In Aruba's case, 'use the source' seems like a bad advice at the moment :smile:.